clients/admin-ui/src/types/react-table-config.d.ts

```diff
@@ -0,0 +1,127 @@
+import {
```

Comment:
Side note for the future: The next major version of react table was rewritten in typescript. This would be a nice upgrade to do in the near future. The version we're using, 7.x.x, is EOL.

I created an issue to upgrade this in plus: https://github.com/ethyca/fidesplus/issues/758

Reply:
oh sad! but cool it's rewritten in typescript, will be great to get rid of this declaration file